style.css

```diff
@@ -24,49 +26,6 @@
   background-color: white;
 }
 
-.toolbar {
-  display: flex;
-  flex-direction: row;
-  top: 44px;
-  align-items: center;
-}
-
-img {
-  display: flex;
-  flex-direction: column;
-  align-items: flex-end;
-}
-
-.toolbar img {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-end;
-  position: absolute;
-  width: 18px;
-  height: 12px;
-  right: 24px;
-  top: calc(50% - 40px / 2);
-  margin-top: 35px;
-  margin-right: 24px;
-}
-
-.toolbar a {
-  width: 76px;
-  height: 20px;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 18px;
-  line-height: 20px;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  color: #6070ff;
-  margin-top: 58px;
-  margin-left: 24px;
-  text-decoration: none;
-}
-
 .headerline {
```

Comment:
- [ ] You did an excellent job on this project, but there is still progress to be made, but you are almost there. As we are imitating the figma design, and if we compare your design to  the figma design are not very similar, please fix this issue so that your project looks professional. . :wink:
Your Design 
![go1](https://user-images.githubusercontent.com/88894525/194706765-294d2c88-8106-4d35-906c-b15e13c3686b.png)

Figma 
![arrange2](https://user-images.githubusercontent.com/88894525/194706789-58f15a39-36fd-4452-84f2-59a5c2c1b31b.png)